### PR TITLE
Append to body all selects inside of user table

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
### ☑️ Resolves

* Part of https://github.com/nextcloud/server/issues/36965
* Fixes: NcSelect is broken so far table will be moved to left or right
* After state is a bit better, but don't know how to fix this issue completely. Probably it is ok to leave it like it is now.

### 🖼️ Screenshots

🏚️ Before

Peek 2023-10-11 10-18

 🏡 After

Peek 2023-10-16 11-42

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — CHANGES_REQUESTED** [review]
> Based on your video, the select dropdown still overlaps?

Yes, but not so much as before.

@Pytal do you have some other ideas?

**Comment by Pytal** [comment]
> > Based on your video, the select dropdown still overlaps?
> 
> Yes, but not so much as before.

Reply in the same comment !?🤪

**Comment by Pytal** [comment]
> @Pytal do you have some other ideas?

Pushed a fix :)

**Comment by JuliaKirschenheuter** [comment]
> Pushed a fix :)

Thank you!

**Review by szaimen — COMMENTED** [review]
Still looks a bit weird if there are not enough user rows but otherwise it works fine now :)

image

**Review by susnux — APPROVED** [review]

File: apps/settings/src/components/Users/shared/styles.scss [hunk 1]
```diff
@@ -38,6 +38,7 @@
 		min-width: var(--cell-width);
 		width: var(--cell-width);
 		color: var(--color-main-text);
+		border-bottom: 1px solid var(--color-border);
```
Comment (ShGKme) [inline apps/settings/src/components/Users/shared/styles.scss:41]: This adds an unwanted bottom border to the users count in the end:

image

**Review by ShGKme — CHANGES_REQUESTED** [review]
Works with select great now, but there is an extra border in the table footer now now

**Review by Pytal — APPROVED** [review]

**Review by ShGKme — APPROVED** [review]

**Pull request merged**